Update docs for pdc+sigv4 (#110787)
grafana/grafana: docs/sources/datasources/prometheus/configure/_index.md

- **Private data source connect** - _Only for Grafana Cloud users._ Private data source connect, or PDC, allows you to establish a private, secured connection between a Grafana Cloud instance, or stack, and data sources secured within a private network. Click the drop-down to locate the URL for PDC. For more information regarding Grafana PDC refer to [Private data source connect (PDC)](ref:private-data-source-connect) and [Configure Grafana private data source connect (PDC)](https://grafana.com/docs/grafana-cloud/connect-externally-hosted/private-data-source-connect/configure-pdc/#configure-grafana-private-data-source-connect-pdc) for steps on setting up a PDC connection.

>   If PDC is used with SIGv4 (AWS Signature Version 4 Authentication) the PDC agent must allow internet egress to`sts.<region>.amazonaws.com:443`.

Reviewer: ```suggestion
  If you use PDC with SIGv4 (AWS Signature Version 4 Authentication), the PDC agent must allow internet egress to`sts.<region>.amazonaws.com:443`.
```

>   Click **Manage private data source connect** to be taken to your PDC connection page, where you’ll find your PDC configuration details.

Reviewer: ```suggestion
  Click **Manage private data source connect** to open your PDC connection page and view your configuration details.
```
